frontend/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
This file can be removed if not used

Reply:
This will be used later for performance metrics of website. This is the reason I didn't removed it
Thanks